src/functions.jl

```diff
@@ -63,31 +63,30 @@ end
 
 Returns the (Q/I, U/I, V/I) fractional polarization vector as a 3-element static vector.
 """
-fracpolarization(s::StokesParams{T}) where {T} = polarization(s)*inv(s.I + eps(T))
-fracpolarization(s::StokesParams{Complex{T}}) where {T} = polarization(s)*inv(s.I + eps(T))
+fracpolarization(s::StokesParams) = polarization(s)*inv(s.I)
 
 """
     mpol(m::StokesParameters{<:Real})
 
 
 Compute the complex fractional linear polarization of a Stokes Parameter `m`
 """
-mpol(m::StokesParams{T}) where {T<:Real} = (m.Q + 1im*m.U)/(m.I + eps(T))
+mpol(m::StokesParams) = (m.Q + 1im*m.U)/m.I
```

Comment:
```suggestion
mpol(m::StokesParams) = linearpol(m)*inv(m.I)
```
Might as well clean this up as well.